dbms/src/Storages/DeltaMerge/RowKeyRange.h

```diff
@@ -33,13 +33,25 @@ using HandleValuePtr = std::shared_ptr<String>;
 struct RowKeyRange;
 using RowKeyRanges = std::vector<RowKeyRange>;
 
-inline int compare(const char * a, size_t a_size, const char * b, size_t b_size)
+namespace
```

Comment:
Better to define an explicit name for this namespace because it is in a `.h` but not `.cpp` file. Maybe `CommonHandle`